src/PhpSpreadsheet/Writer/Xlsx/Rels.php

```diff
@@ -432,4 +436,25 @@ private function writeRelationship(XMLWriter $objWriter, $pId, $pType, $pTarget,
             throw new WriterException('Invalid parameters passed.');
         }
     }
+
+    /**
+     * @param $objWriter
+     * @param \PhpOffice\PhpSpreadsheet\Worksheet\Drawing $drawing
+     * @param &$i
+     *
+     * @throws WriterException
+     */
+    private function writeDrawingHyperLink($objWriter, $drawing, &$i)
```

Comment:
Parameters should be typed and `&$i` should be avoided and instead use return value.